aeron-driver/src/main/c/media/aeron_udp_channel_transport.c

```diff
@@ -226,7 +226,7 @@ int aeron_udp_channel_transport_init(
     if(set_socket_non_blocking(transport->fd) < 0)
```

Comment:
Minor point on formatting here. No space after if.

Reply:
Ok, I'll fix that in the next PR